dask/dataframe/core.py

```diff
@@ -333,12 +333,22 @@ def iloc(self):
         # see https://github.com/ContinuumIO/dask/pull/507
         raise AttributeError("Dask Dataframe does not support iloc")
 
-    def repartition(self, divisions):
+    def repartition(self, divisions, force=False):
         """ Repartition dataframe along new divisions
 
+        Parameters
+        ----------
+
+        divisions: list
+            List of partitions to be used
+        force: bool, default False
+            Whether to allow expand the existing divisions.
```

Comment:
Suggest add contract alongside expand.  Typo in uppoer.

Suggest the following text:

Allows the expansion or contraction of the existing divisions
If False then the new divisions lower and upper bounds must be the same as the old divisions


Allows the new divisions to have different minimum/maximum values from the old divisions.